sunpy/time/time.py

```diff
@@ -195,6 +195,19 @@ def convert_time_astropy(time_string, **kwargs):
     return time_string
 
 
+@convert_time.register(list)
+def convert_time_list(time_list, format=None, **kwargs):
+    item = time_list[0]
+    # If we have a list of strings, need to get the correct format from our
+    # list of custom formats.
+    if isinstance(item, str) and format is None:
+        format = _get_time_fmt(item)
+        return Time.strptime(time_list, format, **kwargs)
```

Comment:
```suggestion
        string_format = _get_time_fmt(item)
        return Time.strptime(time_list, string_format, **kwargs)
```
I suggest using a different name for the variable to avoid confusing future maintainers